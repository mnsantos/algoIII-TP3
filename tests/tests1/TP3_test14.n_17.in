17 55
1 10
4 10
1 17
5 10
3 12
15 16
14 15
13 15
12 14
12 13
11 13
8 15
8 16
9 16
10 17
3 7
1 2
1 8
1 15
1 11
6 10
6 13
1 7
4 15
1 4
2 15
1 13
6 11
7 10
4 14
1 3
5 8
2 9
7 15
2 6
4 17
2 12
3 8
5 15
10 11
8 10
4 5
8 12
1 9
1 6
1 12
1 14
11 12
2 3
7 12
6 8
1 5
4 16
3 16
1 16
0
