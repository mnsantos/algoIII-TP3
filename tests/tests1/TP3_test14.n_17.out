27 3 1 8 15
